pom.xml

```diff
@@ -209,6 +209,8 @@
 			<plugin>
 				<groupId>io.spring.javaformat</groupId>
 				<artifactId>spring-javaformat-maven-plugin</artifactId>
+				<!-- Remove version when : https://github.com/spring-cloud/spring-cloud-build/pull/235 is integrated and released -->
```

Comment:
I will take care of this one also, when it is fixed over there

Reply:
Let me know when the PR is ready and we can do this all at once...or you can separate this change out into a separate PR

Reply:
Right, this is ready. Let me know when you close and release that one, and Ill integrate it in after after that. I hope I understood you correctly

Reply:
@ryanjbaxter build has passed over there, FYI

Reply:
Its merged, the build should only take a few minutes before snapshots are published